### PR TITLE
e2e: Support running pruntime in Gramine

### PR DESCRIPTION
**Pull request by kvinwang**

### CODE REVIEW

File: e2e/src/fullstack.js [hunk 1]
```diff
@@ -1241,25 +1244,35 @@ function newPRuntime(teePort, tmpPath, name = 'app') {
...
-            }
-        });
+        if (inSgx) {
+            fs.cpSync(pRuntimeDir, workDir, { recursive: true })
+            fs.mkdirSync(path.resolve(`${sealDir}/protected_files/`), { recursive: true });
+            fs.mkdirSync(path.resolve(`${sealDir}/storage_files/`), { recursive: true });
+        } else {
+            fs.mkdirSync(sealDir, { recursive: true });
+            const filesMustCopy = ['Rocket.toml', pRuntimeBin];
+            const filesShouldCopy = ['GeoLite2-City.mmdb']
```
Comment (h4x3rotab) [inline e2e/src/fullstack.js:1254]: Let's file an issue to remove GeoLite access completely. We are going to use proactive network prob to replace geolocation.

**Review by h4x3rotab — COMMENTED** [review]
Readability looks good. Didn't dig into how Gramine is executed though.

File: e2e/src/fullstack.js (hunk 1 above)
Comment (kvinwang) [reply e2e/src/fullstack.js:1254]: Let me just do it now.

Comment (kvinwang) [reply e2e/src/fullstack.js:1254]: https://github.com/Phala-Network/phala-blockchain/pull/995

**Review by h4x3rotab — APPROVED** [review]

**Pull request merged**